components/NavigationBar/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import Link from 'next/link';
+
+const NavigationBar = styled.nav`
+  display: flex;
+  justify-content: center;
+  background-color: #f1f1f1;
+  padding: 10px;
+`;
+
+const NavItem = styled.div`
```

Comment:
a paragraph would be more semantic here:

```suggestion
const NavItem = styled.p`
```